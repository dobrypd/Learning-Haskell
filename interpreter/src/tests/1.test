int a = 1;
int b = 12;
int c = 123;
int d = 1234;
bool x = true;
bool y = false;
bool z = true;
float i = -0.1;
float j = -0.01;
float k = -0.001;
