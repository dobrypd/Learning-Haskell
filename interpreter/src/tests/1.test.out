Welcome to Kappa interpreter
"Function main does not exist!"
[(fromList [(Ident "a",1),(Ident "b",12),(Ident "c",123),(Ident "d",1234),(Ident "i",-0.1),(Ident "j",-1.0e-2),(Ident "k",-1.0e-3),(Ident "x",True),(Ident "y",False),(Ident "z",True)],fromList [],fromList [])]
